libc-bottom-half/crt/crt1.c

```diff
@@ -3,6 +3,7 @@ extern void __wasm_call_ctors(void);
 extern int __original_main(void);
 extern void __prepare_for_exit(void);
 
+__attribute__((export_name("_start")))
 void _start(void) {
```

Comment:
Remind my why this file is duplicated here and in basics/crt/crt1.c?

Reply:
Good point; it's an artifact of the "reference sysroot" idea that isn't important anymore. I'll submit a separate PR to remove it.